quantum-adapters/kiwoom-adapter/src/kiwoom_api/functions/new_stock_rights.py

```
@@ -0,0 +1,199 @@
+    from ..auth.token_manager import token_manager
+    from ..models.chart import NewStockRightsResponse, StructuredNewStockRightsData, NewStockRightsApiResponse, NewStockRightsData
+except ImportError:
+    import sys
+    from pathlib import Path
+    src_path = Path(__file__).parent.parent.parent
+    if str(src_path) not in sys.path:
+        sys.path.insert(0, str(src_path))
+    
+    from kiwoom_api.config.settings import settings
+    from kiwoom_api.auth.token_manager import token_manager
+    from kiwoom_api.models.chart import NewStockRightsResponse, StructuredNewStockRightsData, NewStockRightsApiResponse, NewStockRightsData
```

Ruff (0.12.2)

`kiwoom_api.models.chart.NewStockRightsResponse` imported but unused

Remove unused import

(F401)